Api/REST.md

```diff
@@ -210,6 +224,8 @@ Other codes that are commonly used are:
 
 *`CREATED (201)`* - resource is created successfully.
 
+*`NO CONTENT (204)`* - resource is updated or deleted successfully.
```

Comment:
Should we maybe emphasize that, as obvious as it sounds, the response body is empty when using this response code? Asking because this definition could be used for the status code 200 as well.

Reply:
It does make sense to emphasize that `unlike OK (200), there is no option of adding response payload`.